backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -314,8 +328,8 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
 
     var serviceRequest =
         FhirConverter.convertToServiceRequest(
-            ServiceRequest.ServiceRequestStatus.ACTIVE,
-            testPerformedCode,
+            ServiceRequest.ServiceRequestStatus.COMPLETED,
+            testOrderLoinc,
```

Comment:
Can you give a tl;dr of these changes? Were we simply sending the wrong values before, and did that have any practical implications?

Reply:
I believe so, this is just matching it to what the single entry fhir bundle is doing.